lib/surveyor/models/response_methods.rb

```diff
@@ -8,9 +8,9 @@ module ResponseMethods
 
       included do
         # Associations
-        belongs_to :response_set
-        belongs_to :question
-        belongs_to :answer
+        belongs_to :response_set, required: false
+        belongs_to :question, required: false
+        belongs_to :answer, required: false
```

Comment:
How does a response not require its set, question, and answer?

I get if `required: false` gets the specs running, but is this really correct?

Reply:
It actually is and it's 99% because of the parser. Due to the webbed nature of how these all work, it builds a lot of these models, keeps them in memory, and then wires up the links. It's not obvious but it is necessary for a lot of the more sophisticated stuff that this does to work.